### PR TITLE
Improve UX of default search filter sizes

### PR DESCRIPTION
**Pull request by bencrouse**

### CODE REVIEW

**Review by tubbo — APPROVED** [review]

File: core/config/initializers/00_configuration.rb [hunk 1]
```diff
@@ -127,11 +127,6 @@
   end
 
   fieldset 'Search', namespaced: false do
-    field 'Default Search Facet Result Sizes',
-      type: :integer,
-      default: 10,
-      description: 'The number of filter results returned for each filter type.'
-
     field 'Search Facet Result Sizes',
```
Comment (mttdffy) [inline core/config/initializers/00_configuration.rb:130]: the description of this now doesn't make sense since it references the "default config above"

**Review by mttdffy — CHANGES_REQUESTED** [review]

File: core/config/initializers/00_configuration.rb (hunk 1 above)
Comment (bencrouse) [reply core/config/initializers/00_configuration.rb:130]: In the code "above" refers to the specific filter settings, and it renders that way in the UI too
Screen Shot 2020-09-28 at 8 19 05 AM

Comment (mttdffy) [reply core/config/initializers/00_configuration.rb:130]: Im talking about "search facet result sizes" description, says "the default config above", which is now below it.

Comment (bencrouse) [reply core/config/initializers/00_configuration.rb:130]: Oh derp, I've updated.

**Pull request merged**